pkg/server/storage/mysql.go

```diff
@@ -55,6 +55,42 @@ func (my *MysqlStorage) QueryHostkeys() (ResultIterator, error) {
 	return &SqlResultIterator{Rows: rows}, nil
 }
 
+func (my *MysqlStorage) RecordGitHubMapping(mapping map[string]string) error {
+	// Prepare for batch insert
+	entries := make([]string, 0, len(mapping))
+	values := make([]interface{}, 0, len(mapping)*2)
+	for ssoIdentity, githubUser := range mapping {
+		// Create one set of values for each mapping
+		entries = append(entries, "(?, ?)")
+		// Append matching values for mapping
+		values = append(values, ssoIdentity)
+		values = append(values, githubUser)
+	}
+	// Create query with necessary number of (?, ?) values
+	// Note: The use of VALUES in VALUES(github_username) will be deprecated in MYSQL 8.0.20
```

Comment:
Maybe worth rigth away writing it in a way that won't be deprecated (soon)?

Reply:
I'm going to revisit this, probably going to have different statements for each version because we couldn't find a method that worked on every version